src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -22,107 +24,95 @@ import Icon from '../../../components/Icon';
 import * as Expensicons from '../../../components/Icon/Expensicons';
 import variables from '../../../styles/variables';
 
-class AddPayPalMePage extends React.Component {
-    constructor(props) {
-        super(props);
+const AddPayPalMePage = (props) => {
+    const [payPalMeUsername, setPayPalMeUsername] = useState('');
+    const [payPalMeUsernameError, setPayPalMeUsernameError] = useState(false);
+    const payPalMeInput = useRef(null);
 
-        this.state = {
-            payPalMeUsername: '',
-            payPalMeUsernameError: false,
-        };
-        this.setPayPalMeUsername = this.setPayPalMeUsername.bind(this);
-        this.focusPayPalMeInput = this.focusPayPalMeInput.bind(this);
-    }
+    const growlMessageOnSave = props.translate('addPayPalMePage.growlMessageOnSave');
```

Comment:
NAB, this kind of made me wonder about whether translations should be memoized. I feel like they aren't now (though we will re-render any components that use `withTranslate()` if the language changes) but maybe it would be better to have a `useTranslate('addPayPalMePage.growlMessageOnSave')` that just return the same result unless the language changes. Anyways, not something we should do, but seems like a good use case for hooks.

Reply:
Yeah, I was thinking about that when making this change - memorizing translations sounds like a good idea but maybe something to tackle in a larger separate issue?

Reply:
1000% 👍